admin/class-admin.php

```diff
@@ -99,6 +99,15 @@ function schedule_rewrite_flush() {
 	}
 
 	/**
+	 * Returns all the classes for the admin features
+	 *
+	 * @return array
+	 */
+	public function get_admin_features() {
```

Comment:
This is done so third party users can remove or modify actions that these classes register. We don't use this function, it's meant for third parties.